flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/AutoscalerEventHandlerFactory.java

```diff
@@ -73,6 +75,6 @@ AutoScalerEventHandler<KEY, Context> create(Configuration conf) throws Exception
             AutoScalerEventHandler<KEY, Context> createJdbcEventHandler(Configuration conf)
                     throws Exception {
         var conn = HikariJDBCUtil.getConnection(conf);
-        return new JdbcAutoScalerEventHandler<>(new JdbcEventInteractor(conn));
+        return new JdbcAutoScalerEventHandler<>(new JdbcEventInteractor(conn), Duration.ZERO);
```

Comment:
Why don't get `JDBC_EVENT_HANDLER_TTL` from conf?

Reply:
sorry & thank for the catching.
updated.